### DISCUSSION TITLE
Budget operation

### DISCUSSION THREAD

**Discussion by albertordes** [firefly-iii/firefly-iii#5680] [General]
Hello,

I am starting to use the budget section.

I thought that I set a budget called "Travel" with €1000 per year and the charges that I was adding to said budget were going to be subtracted during that year. Well, when I change the month, I see that in the budget section I have the capital of €1,000 again without the charges of the previous month.

The budget created is without autobudget.

I wanted to know exactly how they work, since I haven't fully understood it.

Thank you very much and greetings

**Comment by JC5** [comment]
How did you set up the yearly amount, exactly?

**Reply by albertordes** [reply]
I added the budget and later on the budget screen I added an amount

This image is from today when the month changes:
![image](https://user-images.githubusercontent.com/70942645/151984010-232a111d-93b8-4129-a7bd-938e48d487a6.png)

This image is changing to january:
![image](https://user-images.githubusercontent.com/70942645/151984227-eb647763-2ad3-49ab-8e9d-92b33543f912.png)

Shouldn't I keep those -€18.15 and keep subtracting from the initial €300 budgeted for the whole year?

Thanks!

**Reply by JC5** [reply]
It should yes, that's a bug that I will pick up in the new layout. Sorry about that.

**Reply by albertordes** [reply]
Ok, thanks for your reply!

**Reply by da4id** [reply]
Hi, do you know when it will be fixed?

**Reply by vitobotta** [reply]
I had opened another thread about this before noticing this one. Looking forward to a fix. Thanks!
